tutorials.md

```diff
@@ -40,3 +44,7 @@ this could be a good point to start!
 #### Simulating Muon Spectroscopy Experiments with MuSpinSim
 * [MuSpinSim usage tutorial](muspinsim/tutorial)
 
+#### Finding the Muon Stopping Site in **[Materials Galaxy](https://materialsgalaxy.stfc.ac.uk/)**
+* [Finding the muon stopping site with `pymuon-suite` in Galaxy](https://bit.ly/Find_Muon_Stopping_Site)
```

Comment:
```suggestion
* [Finding the muon stopping site with `pymuon-suite` in Galaxy](https://gxy.io/GTN:T00402)
```
The GTN provides a purl for the tutorial, it would probably better to use that than rely on an external url shortening service to link to the content.